Source/BugSplatRuntime/Bugsplat_Android_UPL.xml

```diff
@@ -32,6 +53,62 @@
                 mavenCentral()
             }
         </insert>
+
+        <if condition="bUploadSymbols">
+            <true>
+                <log text="BugSplat postprocessing: Start debug symbols upload for Android"/>
+                <insert>
+                    import org.apache.tools.ant.taskdefs.condition.Os
+
+                    def checkExecResult(execResult, failText, standardOutput) {
+                        if (execResult) {
+                            if (execResult.getExitValue() != 0) {
+                                throw new GradleException('Non-zero exit value: ' + execResult.getExitValue())
+                            }
+                            if (standardOutput.toString().contains(failText)) {
+                                throw new GradleException('"' + failText + '" string in output: ' + standardOutput.toString())
+                            }
+                        } else {
+                            throw new GradleException('Returned a null execResult object')
+                        }
+                    }
+
+                    def getUploaderExecName() {
+                        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                            return 'symbol-upload-win.exe'
+                        }
+                        if (Os.isFamily(Os.FAMILY_MAC)) {
+                            return 'symbol-upload-macos'
+                        }
```

Comment:
Does it make sense to add Linux support here too?

Reply:
After looking again, it looks like adding Linux support for uploads is a bit beyond the scope of this PR. At first I thought it might just mean adding a few lines here.